Dockerfile

```diff
@@ -1,34 +1,56 @@
-ARG PROJ_VERSION="7.1.0"
+## Inspired by Docker image `kisiodigital/rust-ci:latest-proj` for `proj` installation
+## See https://github.com/CanalTP/ci-images/blob/master/rust/proj/Dockerfile
```

Comment:
Those could be moved below, on proj-builder

Reply:
Not the `RUNTIME_DEPENDENCIES` which is also used in other stages. And for `PROJ_VERSION`, I think it's pretty nice to have it configurable for `docker build`. I'll move `BUILD_DEPENDENCIES` and transform into a `ENV` (also, `curl` is not needed anymore in `BUILD_DEPENDENCIES`).

Reply:
Done in 1c641b1fcc6be28f7446a7a064050b95988cc4b7.